### PR TITLE
docs-link: increase timeout for the check

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Description

docs-link: increase the timeout for the check

Closes: https://github.com/operator-framework/operator-sdk/issues/4172

### CODE REVIEW

File: hack/check-links.sh [hunk 1]
```diff
@@ -15,4 +15,5 @@ docker run --rm -v sdk-html:/target klakegg/html-proofer:3.18.8 /target \
   --empty-alt-ignore \
   --http-status-ignore 429 \
   --allow_hash_href \
+  --typhoeus "{ 'typhoeus': { 'timeout': 300} }" \
```
Comment (camilamacedo86) [inline hack/check-links.sh:18]:
```suggestion
  --typhoeus '{"followlocation":true,"connecttimeout":10,"timeout":300}' \
```

File: hack/check-links.sh [hunk 2]
```diff
@@ -15,4 +15,5 @@ docker run --rm -v sdk-html:/target klakegg/html-proofer:3.18.8 /target \
   --empty-alt-ignore \
   --http-status-ignore 429 \
   --allow_hash_href \
+  --typhoeus '{"followlocation":true,"connecttimeout":10,"timeout":300}' \
```
Comment (camilamacedo86) [inline hack/check-links.sh:18]:
```suggestion
  --typhoeus '{"followlocation":true,"connecttimeout":10,"timeout":600}' \
```

File: hack/check-links.sh [hunk 3]
```diff
@@ -15,4 +15,5 @@ docker run --rm -v sdk-html:/target klakegg/html-proofer:3.18.8 /target \
   --empty-alt-ignore \
   --http-status-ignore 429 \
   --allow_hash_href \
+  --typhoeus '{"followlocation":true,"connecttimeout":10,"timeout":600}' \
```
Comment (camilamacedo86) [inline hack/check-links.sh:18]:
```suggestion
  --typhoeus '{"followlocation":true,"connecttimeout":1000,"timeout":1000}' \
```

**Review by laxmikantbpandhare — APPROVED** [review]
/lgtm

**Comment by laxmikantbpandhare** [comment]
It looks like `timeout` is the total time for the entire request while `connecttimeout` is the time limit for connection. I hope 1000 for each is enough.

File: hack/check-links.sh [hunk 4]
```diff
@@ -15,4 +15,5 @@ docker run --rm -v sdk-html:/target klakegg/html-proofer:3.18.8 /target \
   --empty-alt-ignore \
   --http-status-ignore 429 \
   --allow_hash_href \
+  --typhoeus '{"followlocation":true,"connecttimeout":1000,"timeout":1000}' \
```
Comment (camilamacedo86) [inline hack/check-links.sh:18]:
```suggestion
  --typhoeus '{"followlocation":true,"connecttimeout":600,"timeout":600}' \
```

**Review by asmacdo — APPROVED** [review]
/lgtm

/hold until the release is done

**Comment by camilamacedo86** [comment]
/hold cancel

Release is done ;-)

**Pull request merged**